setup.cfg

```diff
@@ -5,6 +5,8 @@ doc_files = docs/Makefile docs/source/*.rst
 
 [flake8]
 max-line-length = 120
+per-file-ignores:
+    pyresample/test/*.py:D102
```

Comment:
Do we really need this?

Reply:
I've added it to some other pytroll library after you didn't seem to mind there I also put it here. This basically makes it so we don't have to have a docstring on every test method. They are often redundant and unnecessary.

Reply:
I'm not found of this, but ok, it's friday...

Reply:
We've had multiple discussions on this (I know @gerritholl has been annoyed by it too) where we have methods called `test_resample_geostationary` with a docstring `"Test resampling works with geostationary data."`. The docstring isn't needed except to make flake8 happy.